From: James Page <james.page@ubuntu.com>
Date: Mon, 12 Aug 2013 10:56:00 +0100
Subject: Drop use of org.codehaus.mojo.signature

This creates problems for the Debian version of animal sniffer.
---
 pom.xml | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/pom.xml b/pom.xml
index 8eaecdd..9d54edd 100644
--- a/pom.xml
+++ b/pom.xml
@@ -582,13 +582,6 @@ THE SOFTWARE.
             </goals>
           </execution>
         </executions>
-        <configuration>
-          <signature>
-            <groupId>org.codehaus.mojo.signature</groupId>
-            <artifactId>java15</artifactId>
-            <version>1.0</version>
-          </signature>
-        </configuration>
       </plugin>
 
       <plugin>
